README.md

```diff
@@ -1,39 +1,110 @@
-# React Component Library (RCL)
+# Athenaeum - A React Component Library (RCL)
 
-React Component Library is a tool to help create and maintain robust design systems, allowing a developer to roll out higher quality, more consistent User Interfaces (UI) faster.
-React Component Library borrows its fundamental design concepts & principles from [Atomic Design](http://atomicdesign.bradfrost.com/table-of-contents/).
+The RCL is a tool to help create and maintain robust design systems, allowing a developer to roll out higher quality, more consistent User Interfaces (UI) faster.
 
-## Installation
-`yarn install`
+To see it in action, visit:
+http://rcl.policygenius.com/v4.1.2/index.html
 
-## Workflow
-* **Starting from scratch**  
-`yarn start` which is equivalent to `yarn run styleguide:build && yarn run styleguide`
+The RCL relies heavily on the following technologies. If you want to learn more about them and how to customize/configure the RCL, take a look at the documentation:
 
-* **Start without rebuild**  
-`yarn run styleguide`
+* [React](https://facebook.github.io/react/)
+* [Styleguidist](https://github.com/styleguidist/react-styleguidist)
+* [CSS Modules](https://github.com/css-modules/css-modules)
+* [Post CSS](http://postcss.org/)
+* [Sass](http://sass-lang.com/)
+* [Webpack 2](https://webpack.js.org/)
 
-* **Generating a new component from a template**  
-`yarn run c`
+We use [Babel](https://babeljs.io/) to compile our JS, which allows us to use the latest features of the language.
 
-* **Create new components!**  
-See [Component Design]()
 
+## Using the RCL locally
+1. Clone the repo:
 
-### Testing
-*(WIP)* Running Tests
-`yarn run test`
+  `git clone git@github.com:policygenius/athenaeum.git`
 
+2. Install dependencies
 
-### Component File Structure
+  `yarn`
 
-See more details about Atomic Design.
+3. Start the server
+
+  `yarn start`
+
+
+
+## Using the NPM package
+
+The RCL is published as a package called "Athenaeum", which can be found [here](https://www.npmjs.com/package/athenaeum).
+
+Installing the package:
+
+  `yarn add athenaeum` or `npm install athenaeum`
+
+
+## Using an RCL component:
+
+RCL components are React components, which are exported as a collection of modules you can import/require like you would any other JS dependency:
+
+e.g.
+
+```
```

Comment:
@drewdrewthis I can review the errors and probably just add them as exclusions to our plugin